packages/nextjs/utils/scaffold-eth/networks.ts

```diff
@@ -166,14 +166,13 @@ export const getNetworkDetailsByChainId = (id: number) => Object.values(NETWORKS
  */
 export const getTargetNetwork = () => {
```

Comment:
Try to change the value in `.env` to invalid. All the console becomes red. (It was before your changes too). I think we need to calculate the value once and save it to the global state/context. (probably another pr, but not sure)